.github/workflows/main.yml

```diff
@@ -8,6 +8,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      
+      - run: rustup toolchain install nightly
```

Comment:
I prefer using stable, nightly is cool and has very nice features, but it's not stable